libr/parse/parse.c

```diff
@@ -262,14 +262,22 @@ static int filter(RParse *p, ut64 addr, RFlag *f, char *data, char *str, int len
 			}
 			if (f) {
 				RFlagItem *flag2;
-				flag = r_flag_get_i2 (f, off);
+				flag = r_flag_get_by_spaces (f, off, 4, "symbols", "imports",
+						"relocs", "strings");
+				if (!flag) {
+					flag = r_flag_get_i2 (f, off);
+				}
 				computed = false;
 				if (!flag) {
 					flag = r_flag_get_i (f, off);
 				}
 				if ((!flag || arm) && p->relsub_addr) {
 					computed = true;
-					flag2 = r_flag_get_i2 (f, p->relsub_addr);
+					flag2 = r_flag_get_by_spaces (f, p->relsub_addr, 4, "symbols",
+							"imports", "relocs", "strings");
+					if (!flag2) {
+						flag2 = r_flag_get_i2 (f, p->relsub_addr);
```

Comment:
same: i believe this shouldn't be needed anymore if `r_flag_get_by_spaces` works well, should it?